about/ossm-supported-platforms-configurations.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-supported-platforms-configurations_{context}"]
+= Supported Platforms and Configurations
```

Comment:
```suggestion
= Supported platforms and configurations
```
Same, See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings.